src/it/scala/com/exasol/cloudetl/kinesis/KinesisAbstractIntegrationTest.scala

```diff
@@ -18,11 +19,10 @@ trait KinesisAbstractIntegrationTest extends AnyFunSuite with BeforeAndAfterAll
   val TEST_SCHEMA_NAME = "kinesis_schema"
   var assembledJarName: String = _
 
-  val exasolContainer = new ExasolContainer(
-    ExasolContainerConstants.EXASOL_DOCKER_IMAGE_REFERENCE
-  )
+  val exasolContainer = new ExasolContainer("7.0.4")
```

Comment:
Update travis.yml to download it before tests. There is latest, but good idea to specificly download this version also.
```
before_install:
 - docker pull exasol/docker-db:7.0.3
```